pages/index.js

```diff
@@ -1,3 +1,53 @@
-export default function Home() {
-  return <h1>Willkommen zum 🐙-Capstone-Template!</h1>;
+import styled from "styled-components";
+import Link from "next/link";
+
+export default function Home({loginName, setLoginName}) {
+  return (
+    <ContainerDiv>
+      <ContainerHeadingOne>Login</ContainerHeadingOne>
+      <ContainerForm>
+        <ContainerInput
+          placeholder={loginName}
+          type="text"
+          onChange={e => {
+            setLoginName(e.currentTarget.value);
+          }}
+        />
```

Comment:
You change the loginName state variable in `_app.js` on every keystroke - this is probably not what you want: You should use an internal state (i.e. in this file) to handle the current value of the input. The callback prop should only be called when the form is submitted.

Reply:
This is a login state which is used among multiple pages. It can't just stay in the login page. Prop drilling is not a concept of next.js, so the useState goes in _app.js for further use.